src/Api/Serializer/ForumSerializer.php

```diff
@@ -60,6 +60,14 @@ public function getId($model)
      */
     protected function getDefaultAttributes($model)
     {
+        $ssoProviders = array_map(function ($provider) {
+            return $this->app->make($provider)->meta();
+        }, $this->app->make('flarum.auth.supported_drivers'));
+
+        $ssoProviders = array_filter($ssoProviders, function ($provider) {
+            return $this->settings->get('auth_driver_enabled_'.$provider);
```

Comment:
It would be better to check whether the provider is enabled **before** instantiating it through the container.

Wouldn't this logic be better placed elsewhere ? Surely it's not the only place that must obtain the list of enabled drivers ? I might have missed another place.

Reply:
I changed it to filter before making. This is, actually, the only place where that's used, so IMO it's alright to keep it here until we'd need it elsewhere.